mne_bids/read.py

```diff
@@ -328,7 +328,7 @@ def _handle_channels_reading(channels_fname, bids_fname, raw):
     return raw
 
 
-def read_raw_bids(bids_fname, bids_root, extra_params=None,
+def read_raw_bids(bids_basename, bids_root, extra_params=None, kind=None,
```

Comment:
@sappelhoff @jasmainak wdyt? breaking backward compat is ok?

if not we need a proper deprecation cycle

Reply:
from my side ok to break -> as far as I can see it, most people who use this are active contributors anyway (at least currently).

Reply:
yeah okay with breaking backward compatibility. I want to know who the users are ... :)